Cargo.toml

```diff
@@ -27,6 +23,9 @@ futures = "0.3.30"
 hex = "0.4.3"
 jsonrpsee = { version = "0.24.7" }
 miette = { version = "7.1.0", default-features = false }
+# Default features include TLS stuff, which on Linux requires OpenSSL. 
+# We only support Bitcoin Core over HTTP, so need for that. 
```

Comment:
You don't need a comment for this. Disabling default features is always fine if it works